QuickNoteVault.API/QuickNoteVault.DAL/UOW/UnitOfWork.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using QuickNoteVault.DAL.Repository;
+
+namespace QuickNoteVault.DAL.UOW
+{
+    public class UnitOfWork : IUnitOfWork
+    {
+        private readonly DbContext _context;
+        private readonly IDictionary<Type, object> _repositories = new Dictionary<Type, object>();
+        private bool _disposed = false;
+
+        public UnitOfWork(DbContext context)
```

Comment:
Change `UnitOfWork(DbContext context)` to `UnitOfWork(ApplicationDbContext context)`